spec/models/token_spec.rb

```diff
@@ -91,10 +94,13 @@
   end
 
   context "configuration" do
-    before{ allow(token).to receive(:jwt_config).and_return({ 'secret_key' => '123456', 'hmac_sha_algo' => 'SECURE', 'issuer' => 'ME' }) }
-    it{ expect( token.send(:secret_key) ).to eql( '123456' ) }
-    it{ expect( token.send(:hmac_sha_algo) ).to eql( 'SECURE' ) }
-    it{ expect( token.send(:issuer) ).to eql( 'ME' ) }
+    before do
+      allow(token).to receive(:jwt_config).
+      and_return( { "secret_key" => "123456", "hmac_sha_algo" => "SECURE", "issuer" => "ME" } )
```

Comment:
Line is too long. [95/80]<br>Redundant curly braces around a hash parameter.<br>Space inside parentheses detected.